usdigitalresponse/usdr-gost
# Grant digest email adjustments

### Ticket #2353 
## Description
Tweaks the grants digest email to bring it fully in line with design expectations. Specifically: 

1. Left-align the product name with the USDR logo
2. Logo clicks through to grant finder tool
3. Add a "from" name
4. Add preheader preview text for the email

## Screenshots / Demo Video

<img width="1878" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/126250/b08a3363-97ab-4002-8c1a-9ebdbea7739f">

Note that the ethereal email tool doesn't display preview text in the inbox, so we'll want to double-check this functionality with real emails sent on staging. 

## Testing

Testing this in dev is a lot of pretty fiddly effort, but here are the steps: 

1. Set up so emails actually get sent/captured in dev
   1. Create an ethereal email to work with: https://ethereal.email/ (nice, one-click creation of an ephemeral email account)
   2. Update the `NODEMAILER_*` environment variables in your `packages/server/.env` file from the values listed on your ethereal account
   3. Comment out `secure: true` in our nodemailer setup (Gmail uses a secure connection, but ethereal doesn't): https://github.com/usdigitalresponse/usdr-gost/blob/05f688e75df1e3fbf6698f88b49281dd117eb280/packages/server/src/lib/email/email-nodemailer.js#L36
   4. Rebuild your docker container so it picks up the new env variables (`docker compose down app` and `docker compose up -d`)
2. Trigger the emails manually via a node REPL
   - Digest email
     1. `docker compose exec app yarn node`
     2. ```node
           const emailService = require('./packages/server/src/lib/email');
           const knex = require('./packages/server/src/db/connection')
           const grants = await knex('grants').limit(3)
           emailService.sendGrantDigest({name: 'Test agency', matchedGrants: grants, matchedGrantsTotal: 3, recipients: ['test@example.com'], openDate: '2024-01-01'})
           ```
3. Once the test email is sent, go to your "Messages" tab in your Ethereal mail account, and you should see the sent email listed there for viewing

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
